### DISCUSSION TITLE
This site can’t provide a secure connection

### DISCUSSION THREAD

**Discussion by sdjeff84** [postalserver/postal#3178] [Help with using Postal]
Not able to access web page.
show error :
This site can’t provide a secure connection
152.89... sent an invalid response.
ERR_SSL_PROTOCOL_ERROR
Pls Help.

**Comment by willpower232** [comment]
Are you using caddy? Is it working currently? What is your `postal status`?

**Reply by sdjeff84** [reply]
yes i am using caddy.
NAME              IMAGE                               COMMAND                  SERVICE   CREATED        STATUS        PORTS
postal-smtp-1     ghcr.io/postalserver/postal:3.3.4   "/docker-entrypoint.…"   smtp      29 hours ago   Up 29 hours
postal-web-1      ghcr.io/postalserver/postal:3.3.4   "/docker-entrypoint.…"   web       29 hours ago   Up 29 hours
postal-worker-1   ghcr.io/postalserver/postal:3.3.4   "/docker-entrypoint.…"   worker    29 hours ago   Up 29 hours

**Reply by willpower232** [reply]
If you are running caddy as per the documentation, you will need to `docker ps -a` and `docker logs postal-caddy` to see what it is doing.

**Reply by sdjeff84** [reply]
62ea816fcc25   caddy                               "caddy run --config …"   31 hours ago   Up 31 hours                                          postal-caddy

**Reply by sdjeff84** [reply]
> If you are running caddy as per the documentation, you will need to `docker ps -a` and `docker logs postal-caddy` to see what it is doing.

pls help.
Waiiting for your reply.

**Reply by willpower232** [reply]
Did you check `docker logs postal-caddy` to see if it had any problems?

**Reply by sdjeff84** [reply]
pls review

{"level":"info","ts":1729311345.8310204,"msg":"using config from file","file":"/etc/caddy/Caddyfile"}
{"level":"info","ts":1729311345.831861,"msg":"adapted config to JSON","adapter":"caddyfile"}
{"level":"warn","ts":1729311345.8318753,"msg":"Caddyfile input is not formatted; run 'caddy fmt --overwrite' to fix inconsistencies","adapter":"caddyfile","file":"/etc/caddy/Caddyfile","line":4}
{"level":"info","ts":1729311345.8325388,"logger":"admin","msg":"admin endpoint started","address":"localhost:2019","enforce_origin":false,"origins":["//localhost:2019","//[::1]:2019","//127.0.0.1:2019"]}
{"level":"info","ts":1729311345.8326144,"logger":"http.auto_https","msg":"server is listening only on the HTTPS port but has no TLS connection policies; adding one to enable TLS","server_name":"srv0","https_port":443}
{"level":"info","ts":1729311345.8326213,"logger":"http.auto_https","msg":"enabling automatic HTTP->HTTPS redirects","server_name":"srv0"}
{"level":"info","ts":1729311345.8328085,"logger":"http","msg":"enabling HTTP/3 listener","addr":":443"}
{"level":"info","ts":1729311345.8328462,"msg":"failed to sufficiently increase receive buffer size (was: 208 kiB, wanted: 7168 kiB, got: 416 kiB). See https://github.com/quic-go/quic-go/wiki/UDP-Buffer-Sizes for details."}
{"level":"info","ts":1729311345.8329086,"logger":"http.log","msg":"server running","name":"srv0","protocols":["h1","h2","h3"]}
{"level":"info","ts":1729311345.8329277,"logger":"http.log","msg":"server running","name":"remaining_auto_https_redirects","protocols":["h1","h2","h3"]}
{"level":"info","ts":1729311345.83293,"logger":"http","msg":"enabling automatic TLS certificate management","domains":["mail.proficientnow.org"]}
{"level":"info","ts":1729311345.8332095,"msg":"autosaved config (load with --resume flag)","file":"/config/caddy/autosave.json"}
{"level":"info","ts":1729311345.8332145,"msg":"serving initial configuration"}
{"level":"info","ts":1729311345.833396,"logger":"tls.obtain","msg":"acquiring lock","identifier":"mail.proficientnow.org"}
{"level":"info","ts":1729311345.8356814,"logger":"tls.cache.maintenance","msg":"started background certificate maintenance","cache":"0xc0003c9980"}
{"level":"info","ts":1729311345.8360326,"logger":"tls.obtain","msg":"lock acquired","identifier":"mail.proficientnow.org"}
{"level":"info","ts":1729311345.8362243,"logger":"tls.obtain","msg":"obtaining certificate","identifier":"mail.proficientnow.org"}
{"level":"info","ts":1729311345.8364825,"logger":"tls","msg":"cleaning storage unit","storage":"FileStorage:/data/caddy"}
{"level":"info","ts":1729311345.8366897,"logger":"tls","msg":"finished cleaning storage units"}
{"level":"info","ts":1729311346.1262486,"logger":"http","msg":"waiting on internal rate limiter","identifiers":["mail.proficientnow.org"],"ca":"https://acme-v02.api.letsencrypt.org/directory","account":""}
{"level":"info","ts":1729311346.1262786,"logger":"http","msg":"done waiting on internal rate limiter","identifiers":["mail.proficientnow.org"],"ca":"https://acme-v02.api.letsencrypt.org/directory","account":""}
{"level":"info","ts":1729311346.1263537,"logger":"http","msg":"using ACME account","account_id":"https://acme-v02.api.letsencrypt.org/acme/acct/2008286417","account_contact":[]}
{"level":"info","ts":1729311346.2703307,"logger":"http.acme_client","msg":"trying to solve challenge","identifier":"mail.proficientnow.org","challenge_type":"tls-alpn-01","ca":"https://acme-v02.api.letsencrypt.org/directory"}
{"level":"error","ts":1729311347.2499752,"logger":"http.acme_client","msg":"challenge failed","identifier":"mail.proficientnow.org","challenge_type":"tls-alpn-01","problem":{"type":"urn:ietf:params:acme:error:tls","title":"","detail":"103.211.38.46: remote error: tls: no application protocol","instance":"","subproblems":[]}}
{"level":"error","ts":1729311347.2501345,"logger":"http.acme_client","msg":"validating authorization","identifier":"mail.proficientnow.org","problem":{"type":"urn:ietf:params:acme:error:tls","title":"","detail":"103.211.38.46: remote error: tls: no application protocol","instance":"","subproblems":[]},"order":"https://acme-v02.api.letsencrypt.org/acme/order/2008286417/315025000537","attempt":1,"max_attempts":3}
{"level":"info","ts":1729311348.392261,"logger":"http.acme_client","msg":"trying to solve challenge","identifier":"mail.proficientnow.org","challenge_type":"http-01","ca":"https://acme-v02.api.letsencrypt.org/directory"}
{"level":"error","ts":1729311349.377041,"logger":"http.acme_client","msg":"challenge failed","identifier":"mail.proficientnow.org","challenge_type":"http-01","problem":{"type":"urn:ietf:params:acme:error:unauthorized","title":"","detail":"103.211.38.46: Invalid response from http://mail.proficientnow.org/.well-known/acme-challenge/B1QPJtAjjMhHpQ1-AbagazeigkagXRzR4vIxs1A9uw8: 404","instance":"","subproblems":[]}}
{"level":"error","ts":1729311349.377121,"logger":"http.acme_client","msg":"validating authorization","identifier":"mail.proficientnow.org","problem":{"type":"urn:ietf:params:acme:error:unauthorized","title":"","detail":"103.211.38.46: Invalid response from http://mail.proficientnow.org/.well-known/acme-challenge/B1QPJtAjjMhHpQ1-AbagazeigkagXRzR4vIxs1A9uw8: 404","instance":"","subproblems":[]},"order":"https://acme-v02.api.letsencrypt.org/acme/order/2008286417/315025008057","attempt":2,"max_attempts":3}

**Reply by willpower232** [reply]
The only thing I can imagine is that your server has a firewall blocking ports 80 and 443 so you will need to check that on both your server and with whoever provides your server as they may have separate settings.
